pipelines/skylab/optimus/scripts/docker_build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
should this live with the other docker build scripts (i.e. in the dockers folder instead of the pipelines folder)?

Reply:
probably should go in the dockers/skylab/ folder but we haven't gotten around to refactoring all that just yet